app/src/main/java/de/dhbw/corona_world_app/map/MapData.java

```diff
@@ -0,0 +1,341 @@
+
+public class MapData {
+
+    private static final String TAG = MapData.class.getName();
+
+    private static Map<String, String> ISOCodeToDisplayName;
+
+    String WebViewStart = "<html><head><title>World Map</title><script type=\"text/javascript\" src=\"https://www.gstatic.com/charts/loader.js\"></script><script type=\"text/javascript\">" +
+            "  window.goToStats = (country) => {console.log(country);};" +
+            "  google.charts.load('current', {" +
+            "  'packages':['geochart']," +
+            "  'mapsApiKey': '" + MapsKey.apiKey + "'});" +
+            "  google.charts.setOnLoadCallback(drawWorldMap);" +
+            "  function drawWorldMap() {" +
+            "  var dataTable = new google.visualization.DataTable();" +
+            "  dataTable.addColumn('string', 'Country');" +
+            "  dataTable.addColumn('number', 'Infected Population');" +
+            "  dataTable.addColumn({'type': 'string', 'role': 'tooltip', 'p': {'html': true}});\n" +
+            "  dataTable.addRows([";
+
+    String WebViewEnd = "  ]);" +
+            "  var options = {" +
+            "    colorAxis: {colors: ['#3bff35', '#e31b23']}," +
+            "    backgroundColor: '#22748f'," +
+            "    datalessRegionColor: '#ffffff'," +
+            "    defaultColor: '#f5f5f5'," +
+            "    trigger: 'selection'," +
+            "    resolution: 'countries'," +
+            "    legend: 'none'," +
+            "    tooltip: {isHtml: true}" +
+            "  };" +
+            "  function createCustomHTMLContent(ISOCountryName, ratio, totalHealthy, totalInfected, totalDeaths) {\n" +
+            "  return '<div style=\"padding:5px 5px 5px 5px;\">' +\n" +
+            "      '<table class=\"countryData_layout\">' + '<tr>' +\n" +
+            "      '<td>Infected Population: </td>' +\n" +
+            "      '<td><b>' + Math.round((ratio + Number.EPSILON) * 1000) / 1000 + '%</b></td>' + '</tr>' + '<tr>' +\n" +
+            "      '<td>Total Healthy: </td>' +\n" +
+            "      '<td><b>' + totalHealthy + '</b></td>' + '</tr>' + '<tr>' +\n" +
+            "      '<td>Total Infected: </td>' +\n" +
+            "      '<td><b>' + totalInfected + '</b></td>' + '</tr>' + '<tr>' +\n" +
+            "      '<td>Total Deaths: </td>' +\n" +
+            "      '<td><b>' + totalDeaths + '</b></td>' + '</tr>' + '</table>' + '</div>' + " +
+            "      '<center><button type=\"button\" id=\"goTo\" onClick=\"window.goToStats(\\''+ISOCountryName+'\\')\">Go to statistics page</button></center>'" +
+            "  }" +
+            "  var chart = new google.visualization.GeoChart(document.getElementById('geochart-colors'));" +
+            "  chart.draw(dataTable, options);};</script></head>" +
+            "<body style='margin:0;padding:0;'><div id=\"geochart-colors\" style=\"width: 100%; height: 100%;\"></div></body></html>";
+
+    public String putEntries(List<Country> entryList) {
+        initISOToDisplayMap();
```

Comment:
use display name of enum

Reply:
have to wait for ui-branch to implement enum display names